src/app/wallet/proposals/proposal-details/proposal-details.component.ts

```diff
@@ -0,0 +1,152 @@
+import { Component, Input, OnInit, OnDestroy } from '@angular/core';
+import { MatDialog } from '@angular/material';
+import * as d3 from 'd3';
+import { Log } from 'ng2-logger';
+
+import {
+  ProposalVoteConfirmationComponent
+} from 'app/modals/proposal-vote-confirmation/proposal-vote-confirmation.component';
+import { ProposalsService } from 'app/wallet/proposals/proposals.service';
+import { SnackbarService } from 'app/core/snackbar/snackbar.service';
+import { VoteOption } from 'app/wallet/proposals/models/vote-option.model';
+import { ModalsHelperService } from 'app/modals/modals-helper.service';
+import { Proposal } from 'app/wallet/proposals/models/proposal.model';
+import { ProposalResult } from 'app/wallet/proposals/models/proposal-result.model';
+import { GraphOption } from 'app/wallet/proposals/models/proposal-result-graph-option.model';
+import { VoteDetails } from 'app/wallet/proposals/models/vote-details.model';
+
+@Component({
+  selector: 'app-proposal-details',
+  templateUrl: './proposal-details.component.html',
+  styleUrls: [
+    './proposal-details.component.scss',
+    '../../../../../node_modules/nvd3/build/nv.d3.css'
```

Comment:
The relative path to node_modules is not exactly ideal... Is there not a better way of doing this?

Reply:
@AllienWorks  @kewde @zaSmilingIdiot 
Can I add the `nv.d3.css` file in the same directory: `src/app/wallet/proposals/proposal-details/nv.d3.css` or need to create the `css` directory in the assets and move the  `nv.d3.css` as the `assets/css/nv.d3.css`

Reply:
should be in the `assets/css` directory ? , if needed graph any where in the future.

Reply:
@cis-shubham-t you can put it in `/src/assets/scss` where the rest of styles is. That would be ideal.